config/csv_file_defaults.yml

```diff
@@ -1,5 +1,6 @@
 
 Weam:
-  skip_lines_before_header: 0
-  skip_lines_after_header: 1
-  delimiter: ','
+  skipped_first_lines: 0
+  col_sep: ','
+  row_sep: "\r\n"
```

Comment:
Is there any way to make cvs upload work irrespective of line termination? Or are there CSVs that have `\n` within a line or something?